src/docs/development/tools/ide/vs-code.md

```diff
@@ -143,8 +143,8 @@ To open Observatory:
 Assists are code changes related to a certain code identifier. A number of these
 are available when the cursor is placed on a Flutter widget identifier, as
 indicated by the yellow lightbulb icon. The assist can be invoked by clicking
-the lightbulb, or by using the keyboard shortcut `Ctrl`+`Enter`, as illustrated
-here:
+the lightbulb, or by using the keyboard shortcut `Ctrl`+`.` (`Cmd`+`.` on macOS), as
```

Comment:
macOS -> Mac

(in accordance with the [Doc Style Guide section on UI elements](https://developers.google.com/style/ui-elements?hl=th#keyboard))

Reply:
The guidelines also suggest using the full modifier key name (e.g. `Command`), but I can fix that later, since there are occurrences in other places too.

Reply:
I've changed this, though I think there are other instances of `macOS` too (I think I copied this one from somewhere else for consistency) so may be worth searching them all if you're going to fix up the `Cmd`s etc.